### ISSUE TITLE
URDF not loading

### ISSUE THREAD

**Issue by mvgianel** [chvmp/champ_setup_assistant#8]
The robot model that I am trying to use with the setup assistant but I keep encountering the following error:
![image](https://user-images.githubusercontent.com/23347523/125211466-8b889580-e274-11eb-9a02-1b90d4246d77.png)

My robot has an extra joint compared to the one showed in the example, besides having a hip, knee, and foot, it has an ankle. Could that be causing the error? Or does anyone know what could be causing it?

**Comment by grassjelly** [comment]
Yep, you are right, the assistant assumes that each leg has only 3DOF. It'll fail otherwise.
